src/config.rs

```diff
@@ -2780,6 +2780,59 @@ impl TiKvConfig {
         Ok(())
     }
 
+    // As the init of `logger` is very early, this adjust needs to be separated and called
+    // immediately after parsing the command line.
+    pub fn logger_compatible_adjust(&mut self) {
+        let default_tikv_cfg = TiKvConfig::default();
+        let default_log_cfg = LogConfig::default();
+        if self.log_level != default_tikv_cfg.log_level {
+            println!("deprecated configuration, log-level has been moved to log.level");
```

Comment:
You should not use `println!`.

If the logger is initialized, then you should use `info!` instead, if it's not, `eprintln!` should be used instead.

Reply:
Sorry, I missed email notification. I'll submit another PR to fix this. Thanks!